webapi/src/main/scala/org/knora/webapi/slice/admin/repo/rdf/Vocabulary.scala

```diff
@@ -36,7 +38,18 @@ object Vocabulary {
 
     // user group properties
     val belongsToProject: Iri = Rdf.iri(KnoraAdminPrefixExpansion, "belongsToProject")
+
+    // property not in ontology!
+    val belongsToOntology: Iri = Rdf.iri(KnoraAdminPrefixExpansion, "belongsToOntology")
```

Comment:
I remember you wanted to remove this from the Vocabulary becuase it is just a "virtual" property in a single query.